src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
The other possibility that comes to mind @fffw and @oxtoacart is to add sites to the PAC file that we verify are not blocked and have them go **DIRECT**, i.e. the inverse of what we currently do. That would just offload them back to the browser, which is just insanely optimized, keeping things as light as possible on the Lantern side.